docs/openfaas-pro/retries.md

```diff
@@ -32,6 +32,20 @@ Timings are specified in Golang durations such as `10s` or `1m`.
 * `maxRetryAttempts` is the amount of times to try sending a message to a function before discarding it.
 * `httpRetryCodes` is a comma-separated list of HTTP status codes which the queue worker will retry when received from a function.
 
+## Configure retries per function
+
+The retry configuration can be overridden on a per function basis using annotations on the function:
+
+```bash
+faas-cli store deploy chaos \
+  --annotation com.openfaas.retry.attempts=30 \
+  --annotation com.openfaas.retry.codes=429 \
+  --annotation com.openfaas.retry.min_wait=5s \
+  --annotation com.openfaas.retry.max_wait=1m
+```
```

Comment:
Add the equivalent in stack.yml format too please

Reply:
I added an equivalent example for the stack.yml format:

<img width="709" alt="Screenshot 2023-08-31 at 10 19 27" src="https://github.com/openfaas/docs/assets/16267532/e023530c-1c35-492e-a30d-fe62c06ce23e">